### PR TITLE
APIs to configure external IP

### PR DESCRIPTION
**Pull request by Stell0**
https://github.com/nethesis/dev/issues/5763

### CODE REVIEW

**Comment by Stell0** [comment]
Is it better now?

File: root/var/www/html/freepbx/rest/modules/configuration.php [hunk 1]
```diff
@@ -141,3 +141,44 @@ function setLegacyMode($value) {
...
+*/
+$app->get('/configuration/suggestedip', function (Request $request, Response $response, $args) {
+    $ch = curl_init();
+    curl_setopt($ch, CURLOPT_URL, "http://ifconfig.io/ip");
+    curl_setopt($ch, CURLOPT_RETURNTRANSFER, TRUE);
+    curl_setopt($ch, CURLOPT_TIMEOUT, 4);
+    curl_setopt($ch, CURLOPT_HTTPHEADER, array(
+        "Content-Type: application/json;charset=utf-8",
+        "Accept: application/json;charset=utf-8",
+    ));
```
Comment (DavidePrincipi) [inline root/var/www/html/freepbx/rest/modules/configuration.php:175]: The above headers are misleading: the ifconfig.io services ignores them and return a text/plain payload.

I suppose the withJson() returns a JSON string, whilst I was expecting a JSON object with some attributes...

Comment (Stell0) [reply root/var/www/html/freepbx/rest/modules/configuration.php]: I removed the header, but don't the second part

**Comment by Stell0** [comment]
something like { "suggestedip" : "1.2.3.4" } ?
I'd prefer the string because it's consistent with other APIs

**Pull request merged**